arber-salihu/loGIStics
# Issue 48 - Implement Socket for Retrieving Alert Data on Front End

TEST STEPS:
1. Checkout `issue-48`
2. Run the spring boot application
  - You can use VS Code with the Spring Boot Dashboard Extension
  - Run with Intellij
3. Start the front end with `npm run dev`
4. Navigate to the application and check that alerts are being populated in the alerts info panel
    - There are initially no alerts shown. In the future, we will use a get request to get all stored alerts and display be able to display them. For now, the only alerts being shown are the ones that are being retrieved from the NWS.
    - It may take a while for alerts to show up in the side panel because we only receive info from the NWS once a minute. 
    - There should be a new format to account for the new data structure of alerts.

*note: I disabled some of the tests temporarily until we can mock the WebSocket
